backend/src/main/java/gov/cdc/usds/simplereport/api/model/TestEventExport.java

```diff
@@ -232,7 +220,7 @@ public String getPatientSuffix() {
 
   @JsonProperty("Patient_race")
   public String getPatientRace() {
-    return patient.map(Person::getRace).map(raceMap::get).orElse(raceUnknown);
+    return patient.map(Person::getRace).map(PersonUtils.raceMap::get).orElse(FhirUtils.UNK_CODE);
```

Comment:
Nit: It was a little confusing for me to see `FhirUtils` referenced in what is currently a csv event export.